firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/HttpsUrlConnectionFactory.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Google LLC
```

Comment:
I believe ApkUpdater also uses the openHttpsUrlConnection. If this factory is going to be used in AAB I believe it should be used in the APK updater as well.

Reply:
It is used in the ApkUpdater. You might need to expand that file as it's hidden right now since the diff is so big.

Reply:
Ah thanks! Thats exactly what happened.